--- al-0.9.1/configure.ac~	2004-11-21 14:14:19.325639688 +0100
+++ al-0.9.1/configure.ac	2004-11-21 14:14:56.723954280 +0100
@@ -44,7 +44,7 @@
 AC_PROG_LIBTOOL
 
 dnl ## local checks go here (not yet)
-dnl sinclude(al.ac)
+#
 dnl AL_CHECK_ALL
 
 AC_CHECK_EXTLIB([OSSP ex], ex, __ex_ctx, ex.h,
--- al-0.9.1/Makefile.in~	2002-12-16 15:46:11.000000000 +0100
+++ al-0.9.1/Makefile.in	2004-11-28 02:36:06.000000000 +0100
@@ -71,7 +71,7 @@
 
 $(LIB_NAME): $(LIB_OBJS)
 	@$(LIBTOOL) --mode=link $(CC) -o $(LIB_NAME) $(LIB_OBJS) -rpath $(libdir) \
-	    -version-info `$(SHTOOL) version -l txt -d libtool VERSION`
+	    -version-info `$(SHTOOL) version -l txt -d libtool VERSION` $(LIBS)
 
 $(TST_NAME): $(TST_OBJS) $(LIB_NAME)
 	@$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $(TST_NAME) $(TST_OBJS) $(LIB_NAME) $(LIBS)
@@ -98,7 +98,7 @@
 	$(SHTOOL) install -c -m 755 al-config $(DESTDIR)$(bindir)/
 	$(SHTOOL) install -c -m 644 al.h $(DESTDIR)$(includedir)/
 	$(SHTOOL) install -c -m 644 al.3 $(DESTDIR)$(mandir)/man3/
-	@$(LIBTOOL) --mode=install $(SHTOOL) install -c -m 644 libal.la $(DESTDIR)$(libdir)/
+	@$(LIBTOOL) --mode=install $(SHTOOL) install -c libal.la $(DESTDIR)$(libdir)/
 
 uninstall:
 	@$(LIBTOOL) --mode=uninstall $(RM) $(DESTDIR)$(libdir)/libal.la
